posthog/client.py

```diff
@@ -324,7 +322,7 @@ def execute_with_progress(
         execution_time = perf_counter() - start_time
 
         QUERY_TIMEOUT_THREAD.cancel(timeout_task)
-        timing("clickhouse_sync_execution_time", execution_time * 1000.0, tags=tags)
+        timing("clickhouse_sync_execution_time", execution_time * 1000.0)
```

Comment:
Why are we removing the tags here? Feels like a separate change. 

No strong feelings though and I'll assume you've checked we're not breaking a graph that has a useful alert

Reply:
Yep - this is a breaking change (called out as such in the PR description).

Adding a bunch of high-cardinality tags to statsd randomly _will_ cause influxdb to go down.

Luckily these metrics are consumed by two things right now:
- Internal metrics dashboards - which we should kill ASAP (have it in my backlog)
- Statsd dashboards - team infrastructure isn't monitoring it so this is up to our team.